R/cchart.R

```diff
@@ -308,6 +308,7 @@ CChart <- function(chart.type, x, small.multiples = FALSE,
         attr(result,  "ChartData") <- x # Used by Displayr to permit exporting of the raw data.
     class(result) <- c(class(result), "visualization-selector")
     attr(result,  "ChartSettings") <- chart.settings
+    attr(result, "footerhtml") <- attr(x, "footerhtml", TRUE)
```

Comment:
Is this safe, or do I need to guard against null attributes? My testing suggests it is safe but wanted to be sure.

Reply:
Yep, this is fine. Null attributes isn't a problem